index.html

```diff
@@ -549,6 +549,14 @@
 that the person can easily switch it off. In general, providing [=consent=] should be rare,
 difficult, highly intentional, and temporary.
 
+In some situations, a [=party=] can comply with a requirement to get a [=person=] to perform
```

Comment:
We should probably say that these are legal requirements. But then the rest of the advice here is hard to follow, since the site is legally required to disobey it. Maybe something like:

> There are some laws that require websites to ask their users' consent to store information even when that information doesn't affect those people's privacy. These laws are harmful. Except to comply with legal requirements, sites should not demand their users' attention to answer questions that aren't important enough to justify the cost to those users.